app/src/main/java/com/gathering/android/intro/IntroComposables.kt

```diff
@@ -0,0 +1,55 @@
+package com.gathering.android.intro
+
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.pager.PagerState
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.gathering.android.common.PageIndicatorView
+
+
+@Composable
+@Preview
+fun pageIndicatorPreview() {
+    PageIndicatorView(
+        isSelected = true,
+        selectedColor = Color.Red,
+        defaultColor = Color.Red,
+        defaultRadius = 5.dp,
+        selectedLength = 5.dp,
+        animationDurationInMillis = 1
```

Comment:
animation wouldnt be noticeable to user with this duration. either provide no value or a longer duration like a second or so

Reply:
it is required if i want to create an indicator so I just put in a value that does not really affect my dots:

fun PageIndicatorView(
    isSelected: Boolean,
    selectedColor: Color,
    defaultColor: Color,
    defaultRadius: Dp,
    selectedLength: Dp,
    **animationDurationInMillis: Int,**
    modifier: Modifier = Modifier,
)

Reply:
or its preview. nvm then